pwndbg/gdblib/config.py

```diff
@@ -71,15 +78,32 @@ def get_set_string(self):
         if not pwndbg.decorators.first_prompt:
             return ""
 
-        return "Set %s to %r" % (self.param.docstring, self.param.value)
+        return "Set %s to %r" % (self.param.docstring, self.native_value)
+
+    def get_show_string(self, svalue):
+        """Handles the GDB `show <param>` command"""
+        return "The current value of %r is %r" % (
+            self.param.name,
+            Parameter._value_to_gdb_native(svalue, self.param.param_class),
+        )
 
     @staticmethod
-    def _value_to_gdb_native(value):
+    def _value_to_gdb_native(value, param_class=None):
         """Translates Python value into native GDB syntax string."""
-        # Convert booleans to "on" or "off". Other types pass through normally
+        # Convert booleans to "on" or "off".
         if isinstance(value, bool):
             return "on" if value else "off"
-
+        elif value is None and param_class == gdb.PARAM_AUTO_BOOLEAN:
+            # None for gdb.PARAM_UINTEGER means "auto".
```

Comment:
`gdb.PARAM_AUTO_BOOLEAN`

Reply:
Ahh, I forgot to modify it after copy and paste🤣
I'll fix this!